src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -117,6 +117,8 @@ internal partial class RecordOfStringAndAny
         public string? Namespace => TryGetValue("x-namespace", out object? value) ? value?.ToString() : null;
         public string? Usage => TryGetValue("x-csharp-usage", out object? value) ? value?.ToString() : null;
 
+        public bool IsEmbeddingsVector => TryGetValue("x-ms-embedding-vector", out var value) && Convert.ToBoolean(value);
```

Comment:
Is it swagger-only?

Reply:
For now yes until this is resolved https://github.com/Azure/typespec-azure/issues/3781